flake.nix

```diff
@@ -95,4 +95,16 @@
         packages = topLevel.bitte-packages;
         legacyPackages = topLevel;
       }));
+
+  nixConfig = {
```

Comment:
This makes it very conventient for non-nix users to use the binary caches. 
They are basically setup automatically when the flake is used for the first time.